src/libstore/include/nix/store/store-api.hh

```diff
@@ -203,7 +203,7 @@ protected:
 
     SharedSync<State> state;
 
-    std::shared_ptr<NarInfoDiskCache> diskCache;
+    std::shared_ptr<NarInfoDiskCache> diskCache = nullptr;
```

Comment:
Hm, this shouldn't be needed since the default constructor for `std::shared_ptr` takes care of this (see https://en.cppreference.com/w/cpp/memory/shared_ptr/shared_ptr).

Reply:
Okay. I was just trying to satisfy gcc and it didn't complain in other instances.